app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -74,10 +73,106 @@ class SubjectAreaDropdown extends React.Component {
       hasReachedMultiselectLimit: false,
     }
 
+    const chooseBorderColorFromProps = (
+      theme,
+      validationStatus = 'default',
+      isFocused,
+    ) =>
+      ({
+        error: theme.colorError,
+        success: theme.colorSuccess,
+        warning: theme.colorWarning,
+        default: isFocused ? '#2684FF' : theme.colorBorder,
+      }[validationStatus])
```

Comment:
Something very similar to this is in #248. Worth pulling out into a separate file?

Reply:
Fair point. I think a basic version would be a good addition to @pubsweet/ui-toolkit but this is sufficiently different.